### DISCUSSION TITLE
Scroling long file names

### DISCUSSION THREAD

**Discussion by Bernie7373** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2828] [General]
Hi there, could it be possible to add scolling long files names in a future release when we want to chose a printing. It would be easier to choose. Thanks

**Comment by radek8** [comment]
Long names are supported. You must also turn on support in Marlin.

**Comment by Bernie7373** [comment]
Hi there,
Thank you for your answer, but I don't know witch ligne to modify. First of
all, is it in the config.ini file ? I tried in line "Marlin Mode Title" to
modify "marlin_title:Marlin Mode" with marlin_title:20, but I have an error.
Could you help me please.
Thank you for your help (and sorry for my English, I'm French).
Bernard

**Comment by radek8** [comment]
Marlin Mode Title
Banner text displayed at the top of the TFT in Marlin Mode.
Value range: [min: 3, max: 20 characters]
marlin_title:Marlin Mode

What do you not understand about this description of value?
20 is only 2 characters. That's why you're getting an error.
This value does not affect long name support.
Here you define the text that will be displayed when you switch to Marli mode.

**Comment by radek8** [comment]
Set in the config.ini file
long_filename:2

You also have to enable Marli in the configuration
LONG_FILENAME_HOST_SUPPORT (in Configuration_adv.h)
and recompile Marlin

**Comment by Bernie7373** [comment]
Hi there,

Thank you very much for your answer and advice. I'm gonna see what I can do
with it.
Best regards.
Bernard GUYOT
